components/team/helpers.ts

```diff
@@ -37,3 +37,13 @@ export const isRoleSuperiorOrSame = (manager: string, target: string) => {
 
   return false
 }
+
+export const createTeamPageLink = (queryParams: Record<string, string>) => {
+  const url = new URL(window.location.href.replace(window.location.search, ''))
+
+  Object.entries({ tab: 'members', ...queryParams }).forEach(([key, value]) => {
+    url.searchParams.set(key, value)
+  })
+
+  return url.toString()
```

Comment:
Did you known ?

```suggestion
  const url = new URL(window.location.href)
  
  url.search = new URLSearchParams(Object.entries({ tab: 'members', ...queryParams })).toString()

  return url.toString()
```

Reply:
yeah but this loses any already existing params that may already exist in the url, right? we want to keep all params and only add the ones that queryParams adds.